File: 01/004/2023_11_03/abc214b.py
S, T = map(int, input().split())

answer = 0
for a in range(0, 101):
    for b in range(0, 101):
        for c in range(0, 101):
            if a + b + c <= S and a * b * c <= T:
                answer += 1

print(answer)

File: 20231214/abc214b.jl
S, T = parse.(Int, split(readline()))

answer = 0

for a in 0:100
    for b in 0:100
        for c in 0:100
            if a+b+c ≤ S && a*b*c ≤ T
                global answer += 1
            end
        end
    end
end

println(answer)
